src/ui/Backdrop/Backdrop.ts

```diff
@@ -56,13 +61,24 @@ export class Backdrop extends Component {
      *
      * The default value is `false`.
      */
-    overlayGradient: ComponentOptions.buildBooleanOption({ defaultValue: false, depend: 'overlayColor' })
+    overlayGradient: ComponentOptions.buildBooleanOption({ defaultValue: false, depend: 'overlayColor' }),
+    /**
+     * If specified, the component will redirect the page to this URL when clicked.
+     */
+    clickUrl: ComponentOptions.buildStringOption(),
```

Comment:
Hum, for result link, it would be much better if it followed the ResultLink component. There is an option (hrefTemplate) which is very flexible. We need the same mechanism here. A simple string URL might work for some case, but is not enough for real deployment scenarios.

Maybe extracting a specific component option for this kind of option (stringTemplateOption maybe? ) could be a good idea.